server/players.js

```diff
@@ -118,9 +118,24 @@ function join(id, displayName)
 	// subscribe client to player-only events
 	player.socket.join(game.id+'_players');
 
+	// determine correct seat for new player
+	// will be the first untaken seat in the order defined
+	var seatPriority = [0,4,8, 2,6,10, 1,5,9, 3,7,11];
+	player.seatNum = seatPriority.find(function(seat){
+		var seatIndex = game.turnOrder.findIndex(p => p.seatNum === seat);
+		return seatIndex < 0;
+	});
+	console.log('seating player at', player.seatNum);
+
 	// add player to the end of the turn order
 	game.turnOrder.push(player);
 
+	// push back czar index if joiner is before them
+	if(placeInTurnOrder <= game.czar){
+		game.czar = (game.czar+1) % game.turnOrder.length;
+		console.log('incrementing czar to', game.czar);
+	}
```

Comment:
Hmm, this turnOrder juggling is confusing to me, so maybe I can't really comment on it, but shouldn't the new `game.czar` state be pushed to the clients somehow?